File: 260123/남녀노소 구분짓기/sex-and-age.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);
        char sex=sc.next().charAt(0);
        int age=sc.nextInt();

        if(sex=='1'){
            if(age>18)  System.out.println("WOMAN");
            else    System.out.println("GIRL");
        }
        else if(sex=='0'){
            if(age>18)  System.out.println("MAN");
            else System.out.println("BOY");
        }
    }
}

File: 260224/남녀노소 구분짓기/sex-and-age.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int gender, age;

    cin>>gender>>age;

    gender==0?(age<19?cout<<"BOY":cout<<"MAN"):(age<19?cout<<"GIRL":cout<<"WOMAN");
    return 0;
}